src/test/java/de/tum/in/www1/artemis/ExamIntegrationTest.java

```diff
@@ -243,4 +243,10 @@ public void testDeleteExam_asInstructor() throws Exception {
         request.delete("/api/courses/" + course1.getId() + "/exams/" + exam1.getId(), HttpStatus.OK);
         verify(examAccessService, times(1)).checkCourseAndExamAccess(course1.getId(), exam1.getId());
     }
+
+    @Test
+    @WithMockUser(value = "admin", roles = "ADMIN")
+    public void DeleteExamThatDoesNotExist() throws Exception {
+        request.delete("/api/courses/" + courseRepo.findAll().get(0).getId() + "/exams/1", HttpStatus.NOT_FOUND);
```

Comment:
Why not chossing a specific course? Like course1 or course2 from line 75 and 76.

Reply:
No real reason, for consistency i replaced the hardcoded value and am now using the id of the specified course.